src/main/java/net/dv8tion/jda/api/audio/AudioReceiveHandler.java

```diff
@@ -39,7 +39,7 @@
      */
     default boolean shouldCombineAudio()
     {
-        return false;
+        return true;
```

Comment:
If default is false then everyone's apps will break and won't receive combined audio